pyproject.toml

```
@@ -0,0 +1,124 @@
+[build-system]
+requires = ['setuptools', 'wheel']
+build-backend = "setuptools.build_meta"
+
+[project]
+name = 'sdmetrics'
+description = 'Metrics for Synthetic Data Generation Projects'
+version = '0.13.1.dev0'
+authors = [{ name = "MIT Data To AI Lab", email = "dailabmit@gmail.com" }]
+license = { text = 'MIT license' }
+classifiers = [
+    'Development Status :: 2 - Pre-Alpha',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: MIT License',
+    'Natural Language :: English',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+]
+readme = 'README.md'
+requires-python = ">=3.8,<3.12"
+keywords = ['sdmetrics', 'sdmetrics', 'SDMetrics']
+dependencies = [
+    "numpy>=1.20.0,<2;python_version<'3.10'",
+    "numpy>=1.23.3,<2;python_version>='3.10'",
+    "pandas>=1.1.3;python_version<'3.10'",
+    "pandas>=1.3.4;python_version>='3.10' and python_version<'3.11'",
+    "pandas>=1.5.0;python_version>='3.11'",
+    "scikit-learn>=0.24,<2;python_version<'3.10'",
+    "scikit-learn>=1.1.3,<2;python_version>='3.10'",
```

I think python 3.10 supports scikit-learn 1.1.0 so if we wanted to be as flexible as possible we can do
```suggestion
    "scikit-learn>=0.24,<2;python_version<'3.10'",
    "scikit-learn>=1.1.0;python_version>='3.10' and python_version<'3.11'",
    "scikit-learn>=1.1.3,<2;python_version>='3.11'",
```